platform-sdk/swirlds-common/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ testModuleInfo {
     requires("com.swirlds.base.test.fixtures")
     requires("com.swirlds.config.api.test.fixtures")
     requires("com.swirlds.config.extensions")
+    requires("com.swirlds.platform.core")
```

Comment:
common requires core? that does not sound right...

Reply:
In the Tests and TestFixtures it does.   The `RandomAddressBookGenerator` [https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/RandomAddressBookGenerator.java](link) lives in the common `TestFixtures` and that has references to `Address` and `AddressBook` in platform-core. 

As long as Common Main does not have a dependency Platform Core, I can understand the value of Common Test having a dependency on modules that rely on Common Main.  It makes it easier to use the code in testing that relies on the code being tested.